### PR TITLE
Adding an example of a global function.

### PR DESCRIPTION
**Pull request by abayer**
cc @reviewbybees

### CODE REVIEW

**Comment by rtyler** [comment]
:bee:

**Comment by varmenise** [comment]
:bee:

File: global-library-examples/global-function/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!groovy
+
+// Loads the standardBuild method from workflowLibs.git/vars/standardBuild.groovy
```
Comment (daniel-beck) [inline global-library-examples/global-function/Jenkinsfile]: It's a *step*, not a method.

Comment (tfennelly) [inline global-library-examples/global-function/Jenkinsfile]: Or global "function"?

Comment (abayer) [inline global-library-examples/global-function/Jenkinsfile]: Reworded to make everyone happy! =)

**Pull request merged**